src/content/docs/infrastructure/host-integrations/host-integrations-list/jmx-monitoring-activemq.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: JMX monitoring with ActiveMQ
+tags:
+  - Integrations
+  - On-host integrations
+  - On-host integrations list
+  - ActiveMQ dashboard
+  - ActiveMQ with New Relic
+metaDescription: 'Get a tour of our quickstarts and dashboards that take JMX-scraped data from your favorite automation software and houses it in the New Relic platform.'
```

Comment:
i'd remove 'quickstarts' language from this. the word 'quickstart' isn't being well used lately. it never was defined well. we should tend to just talk about integrations, similar to our other integrations.

Reply:
@zuluecho9 do you have any strong opinions about differentiating dashboards and integrations? sometimes what's being "integrated" is a series of queries specifically for dashboards. follow up Q, (and maybe we can talk about this more on zoom) should i only default to "integrations" for all of this and refer to dashboards as just the visual graphs that appear in new relic?

Reply:
i think calling these things just 'dashboards' is fine, and maybe calling this an 'ActiveMQ integration' is fine, also, as it NR integrating with ActiveMQ. the thing that is weird/badly used is 'quickstart'; that's mainly what I've avoided, as it hasn't been well defined. And shawn K told me to avoid mostly. 

let me know if you want a have a convo about 'integrations and 'dashboards' language; it is I think an important conversation with a lot of nuance/history (one weirdness is that marketing has used 'dashboard' to only refer to dashboards built from customer queries, and not to pre-built curated UI dashboards, which is very weird to me).